.github/workflows/test-connect.yml

```diff
@@ -1,4 +1,4 @@
-name: "[Test] connect"
+name: "[Test] connect unit"
```

Comment:
these tests arent pure unit tests. I would describe them as e2e tests of connect core.